### PR TITLE
Feature/css layout

### PR DESCRIPTION
**Pull request by nicolas-postigo**

### CODE REVIEW

**Comment by simonbesteiro** [comment]
@nico18las18 this branch has conflicts, resolve them

File: staff/nicolas-postigo/challenges/calculator-app/calculator-app/README.md [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# Calculator App
```
Comment (manuelbarzi) [inline staff/nicolas-postigo/challenges/calculator-app/calculator-app/README.md:1]: this file should not be modified in this pr

File: staff/nicolas-postigo/css-layout/css-layout.css [hunk 2]
```diff
@@ -0,0 +1,109 @@
+body {
```
Comment (manuelbarzi) [inline staff/nicolas-postigo/css-layout/css-layout.css:1]: rename this file into style.css

File: staff/nicolas-postigo/css-layout/css-layout.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
```
Comment (manuelbarzi) [inline staff/nicolas-postigo/css-layout/css-layout.html:1]: rename this file into inde.html

**Comment by manuelbarzi** [comment]
sorry, nico, your pr is a bit caotic! .P redo the branches and re-pr

**Pull request closed without merging**